### PR TITLE
fix: updated translations

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue Overview

This PR addresses https://github.com/metrotranscom/doorway/issues/595

- [x] This change addresses only certain aspects of the issue

## Description

For this PR, I ran the missing translations script and inputted the missing strings into the first four tabs of [this spreadsheet](https://docs.google.com/spreadsheets/d/1gtsSeMC2UJlJJS9N4GLesUVsy-nY1R_tS4RUAKym9nw/edit#gid=0). I did then have to manually filter them, as some are intentionally untranslated or were partners strings that ended up in public. I then downloaded those sheets as a CSV and inputted those into a new script that calls the google translate API and outputs the translations in the right format.

According to product, Doorway will be reviewing the strings at a later time.

I ✨ really ✨ wanted to be able to pull in the existing google translate service but there was a lot of setup to pull the pieces in from the backend.

## How Can This Be Tested/Reviewed?

If you're curious, you can run the script locally with the instructions in the file and make your own test CSV file of English strings. There are a few more changes in the translations files beyond the changed strings because of alpha sorting the full files.

From the initial ticket, there are a few translations that were logged as missing that are actually database translations that do have translations already. I'm not sure why those were logged as missing. We may need more testing of the database strings beyond these missing strings.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: shared-helpers/src/locales/es.json [hunk 1]
```diff
@@ -336,59 +356,66 @@
...
   "application.review.demographics.spokenLanguageLabel": "¿Qué idioma se habla más en su casa? Seleccione una:",
   "application.review.demographics.spokenLanguageOptions.chineseCantonese": "Chino cantonés",
   "application.review.demographics.spokenLanguageOptions.chineseMandarin": "Chino mandarín",
   "application.review.demographics.spokenLanguageOptions.english": "Inglés",
+  "application.review.demographics.spokenLanguageOptions.filipino": "filipino",
+  "application.review.demographics.spokenLanguageOptions.korean": "coreano",
+  "application.review.demographics.spokenLanguageOptions.notListed": "No enlistado",
   "application.review.demographics.spokenLanguageOptions.russian": "Ruso",
   "application.review.demographics.spokenLanguageOptions.spanish": "Español",
+  "application.review.demographics.spokenLanguageOptions.vietnamese": "vietnamita",
```
Comment (mcgarrye) [inline shared-helpers/src/locales/es.json:418]: These should be capitalized

File: shared-helpers/src/locales/es.json [hunk 2]
```diff
@@ -397,16 +424,24 @@
...
...de vivienda no puede verificarse, no recibirá la preferencia. Todos los solicitantes se examinarán según se indica en los criterios de selección de residentes del anuncio.\n\n* El hecho de rellenar esta solicitud no le da derecho a una vivienda ni indica que pueda optar a ella. Tenga en cuenta que no puede modificar su solicitud en línea después de enviarla.\n\nAfirmo que tengo al menos dieciocho años de edad y estoy autorizado a enviar la información que permite la identificación personal (IIP) de cualquier miembro del hogar que figure en la solicitud. Doy mi consentimiento, tanto en nombre de los miembros del hogar que figuran en la solicitud como en el mío propio, para que la IIP se transmita al socio profesional o al gobierno local para los fines indicados en las <a className='lined' href='https://mtc.ca.gov/doorway-housing-portal-terms-use' target='_blank'>condiciones de uso</a>. Estoy de acuerdo con las condiciones anteriores y declaro que la información proporcionada anteriormente es verdadera y exacta, y reconozco que cualquier declaración falsa realizada de forma fraudulenta o negligente en esta solicitud puede dar lugar a la eliminación del proceso de solicitud.<br><br>",
```
Comment (mcgarrye) [inline shared-helpers/src/locales/es.json:430]: Can you remove the random `<br><br>` from this

File: shared-helpers/src/locales/es.json [hunk 3]
```diff
@@ -471,22 +511,19 @@
   "authentication.signOut.success": "La sesión de su cuenta ha sido cerrada exitosamente.",
   "authentication.terms.acceptExtended": "He revisado, comprendo y acepto las Condiciones de uso.",
   "authentication.terms.acceptToc": "Acepto los términos del servicio",
+  "authentication.terms.publicAccept": "Debe aceptar los Términos de uso antes de crear una cuenta.",
+  "authentication.terms.publicTerms": "He revisado los <a href='https://mtc.ca.gov/doorway-housing-portal-terms-use' target='_blank'>Términos de uso</a> de este sitio web, tal como se define ese término en los Términos de uso, y acepto cumplir con todos los requisitos descritos en ellos que se relacionan con mi uso personal. Entiendo que si no uso mi cuenta durante tres años, mi cuenta y todos sus datos serán eliminados.",
```
Comment (mcgarrye) [inline shared-helpers/src/locales/es.json:515]: Need `className='lined'` to underline

File: shared-helpers/src/locales/tl.json [hunk 4]
```diff
@@ -424,22 +469,19 @@
   "authentication.signOut.success": "Matagumpay kang nakapag-log out sa iyong account.",
   "authentication.terms.acceptExtended": "Nasuri ko, nauunawaan at sumasang-ayon ako sa Mga Tuntunin ng Paggamit.",
   "authentication.terms.acceptToc": "Tinatanggap ko ang Mga Tuntunin ng Serbisyo",
+  "authentication.terms.publicAccept": "Dapat mong tanggapin ang Mga Tuntunin ng Paggamit bago gumawa ng account.",
+  "authentication.terms.publicTerms": "Sinuri ko ang <a href='https://mtc.ca.gov/doorway-housing-portal-terms-use' target='_blank'>Mga Tuntunin ng Paggamit</a> para sa Website na ito, dahil ang terminong iyon ay tinukoy sa Mga Tuntunin ng Paggamit, at sumasang-ayon na sumunod sa lahat ng mga kinakailangan na inilarawan doon na nauugnay sa aking personal na paggamit. Naiintindihan ko na kung hindi ko gagamitin ang aking account sa loob ng tatlong taon, ang aking account at lahat ng data nito ay tatanggalin.",
```
Comment (mcgarrye) [inline shared-helpers/src/locales/tl.json:473]: Same here

File: shared-helpers/src/locales/vi.json [hunk 5]
```diff
@@ -472,22 +511,19 @@
   "authentication.signOut.success": "Quý vị đã đăng xuất thành công khỏi tài khoản.",
   "authentication.terms.acceptExtended": "Tôi đã đọc, hiểu và đồng ý với Điều khoản sử dụng.",
   "authentication.terms.acceptToc": "Tôi chấp nhận Điều khoản Dịch vụ",
+  "authentication.terms.publicAccept": "Bạn phải chấp nhận Điều khoản sử dụng trước khi tạo tài khoản.",
+  "authentication.terms.publicTerms": "Tôi đã xem lại <a href='https://mtc.ca.gov/doorway-housing-portal-terms-use' target='_blank'>Điều khoản sử dụng</a> của Trang web này vì thuật ngữ đó được định nghĩa trong Điều khoản sử dụng và đồng ý tuân thủ tất cả các yêu cầu được mô tả trong đó liên quan đến việc sử dụng cá nhân của tôi. Tôi hiểu rằng nếu tôi không sử dụng tài khoản của mình trong ba năm, tài khoản của tôi và tất cả dữ liệu trong đó sẽ bị xóa.",
```
Comment (mcgarrye) [inline shared-helpers/src/locales/vi.json:515]: ^^

Comment (mcgarrye) [inline shared-helpers/src/locales/vi.json:515]: Here as well

File: shared-helpers/src/locales/zh.json [hunk 6]
```diff
@@ -470,16 +511,19 @@
   "authentication.signOut.success": "您已成功登出帳戶。",
   "authentication.terms.acceptExtended": "我已閱讀、理解並同意使用條款。",
   "authentication.terms.acceptToc": "我接受服務條款",
+  "authentication.terms.publicAccept": "在创建帐户之前，您必须接受使用条款。",
+  "authentication.terms.publicTerms": "我已阅读本网站的<a href='https://mtc.ca.gov/doorway-housing-portal-terms-use' target='_blank'>使用条款</a>，该术语已在使用条款中定义，并同意遵守其中所述的与我个人使用相关的所有要求。我理解，如果我三年内未使用我的帐户，我的帐户及其所有数据将被删除。",
```
Comment (mcgarrye) [inline shared-helpers/src/locales/zh.json:515]: Lastly

**Review by mcgarrye — COMMENTED** [review]
Noticing a bunch of `\n\n`'s have been messed up through the translation process resulting in ` n\n` or `\n \n`. If you could go through and resolve those that would be great

**Comment by emilyjablonski** [comment]
@mcgarrye Updated! I did only find four strings w the new line issue w a manual scan, but hopefully I found em all

File: shared-helpers/src/locales/es.json [hunk 7]
```diff
@@ -292,6 +310,9 @@
...
   "application.review.confirmation.whatExpectTitle": "Lo que puede esperar a continuación",
+  "application.review.confirmation.whatHappensNext.fcfs": "### ¿Qué sucede a continuación?\n\n* Los solicitantes elegibles serán contactados por orden de llegada hasta que se cubran las vacantes.\n\n* Las preferencias de vivienda, si corresponde, afectarán el orden por orden de llegada. n\n* Si lo contactan para una entrevista, se le pedirá que complete una solicitud más detallada y proporcione documentos de respaldo.",
+  "application.review.confirmation.whatHappensNext.lottery": "### ¿Qué sucede a continuación?\n\n* Una vez que se cierre el período de solicitud, los solicitantes elegibles serán colocados en orden según el orden de clasificación de la lotería.\n\n* Las preferencias de vivienda, si corresponde, afectarán el orden de clasificación de la lotería.\n \n* Si lo contactan para una entrevista, se le pedirá que complete una solicitud más detallada y proporcione documentos de respaldo.",
```
Comment (mcgarrye) [inline shared-helpers/src/locales/es.json:314]: nit: `\n \n`

**Review by mcgarrye — APPROVED** [review]

File: shared-helpers/src/locales/es.json (hunk 7 above)
Comment (emilyjablonski) [reply shared-helpers/src/locales/es.json]: Fixed thank you!

**Pull request merged**